app/Http/Controllers/Webapp/CategoryController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers\Webapp;
+
+use App\Http\Requests;
+use App\Libs\CdsApiDriver;
+use App\Models\Category;
+use App\Models\Post;
+use App\Models\Site;
+use Illuminate\Http\Request;
+
+class CategoryController extends NeedAuthController {
+
+  public function getListCategories(Request $request) {
+    $categories = Category::where('site_id', $request->get('id'))->get();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "site_id" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1192)